plugins/modules/elasticache_parameter_group.py

```diff
@@ -24,7 +24,7 @@
     description:
       - The name of the cache parameter group family that the cache parameter group can be used with.
         Required when creating a cache parameter group.
```

Comment:
@markuman I'm not sure whether this is usually done in community.aws, but I've seen that in other collections:
```suggestion
        Required when creating a cache parameter group.
      - The value C(redis6.x) is supported since community.aws 3.4.0.
```
Does this make sense here, or would it be a huge exception and thus better to not do this?